blocks/breadcrumb/breadcrumb.js

```diff
@@ -0,0 +1,17 @@
+import { decorateIcons } from '../../scripts/lib-franklin.js';
+
+export default async function decorate(block) {
+  block.querySelectorAll('li a').forEach((link, idx, arr) => {
+    if (!idx < arr.length - 1) {
+      return;
+    }
+
+    const icon = document.createElement('span');
+    icon.classList.add('icon');
+    icon.classList.add('icon-arrow-right');
+
+    link.appendChild(icon);
```

Comment:
Just to give an example, with https://github.com/hlxsites/walgreens/pull/23 these lines would be just
```javascript
link.appendChild(span({ class: 'icon icon-arrow-right' }));
```